bowtie/_cli.py

```diff
@@ -656,8 +690,23 @@ def _do_nothing(*args: Any, **kwargs: Any) -> CaseTransform:
     "--fail-fast",
     is_flag=True,
     default=False,
+    callback=_set_max_fail_and_max_error,
     help="Fail immediately after the first error or disagreement.",
 )
+MAX_FAIL = click.option(
+    "-mf",
+    "--max-fail",
+    type=click.IntRange(min=1),
+    callback=_check_fail_fast_provided,
+    help="Fail immediately if N tests fail in total across implementations",
+)
+MAX_ERROR = click.option(
+    "-me",
```

Comment:
I'd be surprised if this works -- short options usually have to be one letter, so for most CLIs `-ab` should be equivalent to `-a -b` -- if this does work, it certainly would break if we added a `-m` option (or a `-f` one) I suspect.

Anyways, I think this probably is rare enough that it's fine to only have the long option, you can just remove these.

Reply:
Will do. Any other changes ?

Reply:
The rest I think looks good to me!

Reply:
I had a doubt though, currently running `bowtie suite -i lua-jsonschema https://github.com/json-schema-org/JSON-Schema-Test-Suite/blob/main/tests/draft7/type.json --max-fail 2` command also outputs `"did_fail_fast": true` in the output json file. Is that correct or do we explicitly keep it only for --fail-fast ?

Reply:
Good question! No we should also call that failing fast! That marker should signal to someone "the report might be truncated -- there were more cases, but we stopped because we reached a maximum number of failing tests" -- if you see a good place to either document that more explicitly or ensure it with a test that would be good too!

Reply:
Do you mean that I should just keep it as `"failing_fast": true` for both --fail-fast and --max-fail 2  😅?

Reply:
We should output `did_fail_fast: true` any time `--max-fail`, `--max-error` *or* `--fail-fast` ended up being triggered (i.e. we exceeded their number).

And no need to rename the field I think if that's what you're asking?

Reply:
I removed the short flags as you suggested. Its mergable now!